module/src/main/java/zipkin2/module/storage/kafka/ZipkinKafkaStorageProperties.java

```diff
@@ -118,6 +126,38 @@ public void setSpanConsumerEnabled(Boolean spanConsumerEnabled) {
     this.spanConsumerEnabled = spanConsumerEnabled;
   }
 
+  public Boolean getAggregationEnabled() {
+    return aggregationEnabled;
+  }
+
+  public void setAggregationEnabled(Boolean aggregationEnabled) {
+    this.aggregationEnabled = aggregationEnabled;
+  }
+
+  public Boolean getTraceSearchEnabled() {
```

Comment:
can we re-use convention or comment with a link to existing variables? https://github.com/openzipkin/zipkin/tree/master/zipkin-server#environment-variables